### PR TITLE
Add paprika to conda-forge

### PR DESCRIPTION
**Pull request by slochower**

### CODE REVIEW

**Comment by slochower** [comment]
@conda-forge/help-python It's unclear to me why the CI builds are failing with `ValueError: Specified folder /home/conda/conda-recipes/meta.yaml does not exist`. Can you point me in the right direction?

File: recipes/paprika/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package:
+  name: paprika
+  version: "0.1.0"
+
+source:
+  path: ../..
```
Comment (chrisburr) [inline recipes/paprika/meta.yaml]: You need to put a URL and sha256 here, see [here for an example](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L13-L22).

Comment (slochower) [reply recipes/paprika/meta.yaml]: Great, thank you. I misread that line and thought the `sha256` only applied to a PyPI release.

Comment (chrisburr) [reply recipes/paprika/meta.yaml]: No problem!

The version in that URL doesn't match, using a jinja2 variable like the example does will help keep everything consistent. The `{{ version }}` gets replaced with the version defined at the top of the file with `{% set version = "0.0.4" %}`

Comment (chrisburr) [reply recipes/paprika/meta.yaml]: Also you don't need the `path` entry at all.

Comment (slochower) [reply recipes/paprika/meta.yaml]: Yes, got it. I wasn't sure if you (i.e., conda-forge) was going to replace the variables or you were just showing how to programmatically write a `meta.yaml`. Will fix in a sec.

Comment (slochower) [reply recipes/paprika/meta.yaml]: Hopefully fixed in 6f94ea7

Comment (slochower) [reply recipes/paprika/meta.yaml]: Any thoughts on `subprocess.CalledProcessError: Command '['/bin/bash', '-e', '/home/conda/feedstock_root/build_artifacts/paprika_1550788710742/work/conda_build.sh']' returned non-zero exit status 1.`?

Looks like it Azure can't find `pip`? 

`/home/conda/feedstock_root/build_artifacts/paprika_1550788710742/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/bin/python: No module named pip`

Comment (chrisburr) [reply recipes/paprika/meta.yaml]: `pip` should be in the `host` requirements.

Comment (slochower) [reply recipes/paprika/meta.yaml]: Thanks again @chrisburr .

Any idea what's up with the Windows builds? https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=15775&view=logs The error is non-obvious to me (looks like something to do with Windows `conda`):
```
  File "C:\Miniconda\lib\site-packages\conda\core\subdir_data.py", line 60, in __call__
    assert channel.subdir
AssertionError
```

However, I should note that we do not internally test on Windows at all, so I'm not confident that our package will work appropriately there in any case.

Also, Travis seems to be perpetually pending.

Comment (chrisburr) [reply recipes/paprika/meta.yaml]: You can ignore Azure win-64, it has known issues (#7715).

The Travis plan conda-forge has can't handle the load unfortunately so the backlog can get pretty bad. It's being worked on so it should improve at some point.

Comment (slochower) [reply recipes/paprika/meta.yaml]: Gotcha. So we should wait for Travis to build before trying to merge?

File: recipes/paprika/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+about:
+  home: https://github.com/slochower/pAPRika
+  license: BSD-3-Clause 
+  summary: pAPRika is a toolkit for setting up, running, and analyzing free energy molecular dynamics simulations.
+
+extra:
+  recipe-maintainers:
+    - slochower
+    - nhenriksen
```
Comment (chrisburr) [inline recipes/paprika/meta.yaml]: @nhenriksen Are you happy to be listed as a maintainer of this recipe?

File: recipes/paprika/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+test:
+  requires:
+    - pytest
+  imports:
+    - paprika
+
+about:
+  home: https://github.com/slochower/pAPRika
+  license: BSD-3-Clause
```
Comment (chrisburr) [inline recipes/paprika/meta.yaml]:
```suggestion
  license: BSD-3-Clause
  license_family: BSD
  license_file: LICENSE
```

File: recipes/paprika/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+test:
+  requires:
+    - pytest
+  imports:
+    - paprika
+
+about:
+  home: https://github.com/slochower/pAPRika
+  license: BSD-3-Clause 
+  summary: pAPRika is a toolkit for setting up, running, and analyzing free energy molecular dynamics simulations.
```
Comment (chrisburr) [inline recipes/paprika/meta.yaml]: Can you add the `doc_url` and `dev_url` fields.

File: recipes/paprika/meta.yaml (hunk 2 above)
Comment (nhenriksen) [reply recipes/paprika/meta.yaml]: yes!

File: recipes/paprika/meta.yaml (hunk 4 above)
Comment (slochower) [reply recipes/paprika/meta.yaml]: Done

Comment (Lnaden) [reply recipes/paprika/meta.yaml]: Noted. Handling full deployment is a bit beyond what we set out to do with the Cookiecutter, and keeping in full sync with c-f, anaconda, and PyPi would be a daunting task and very much beyond what we aim to do, so we may just update our docs better. But yay for the c-f Linter which take care of such things!

Comment (slochower) [reply recipes/paprika/meta.yaml]: Agreed that full compatibility would be daunting and time consuming. Maybe worthwhile to just note check `examples/meta.yaml` for additional requirements or tips, etc.

**Comment by slochower** [comment]
@chrisburr Thanks again for your help. One quick question: I can see builds on anaconda.org (https://anaconda.org/conda-forge/paprika) and they've been up for about fifteen minutes, but I cannot install from the command line with `conda install -c conda-forge paprika`. Just curious what sort of delay I should expect before updating my installation instructions.

**Pull request merged**